src/test/java/com/redislabs/redisgraph/RedisGraphAPITest.java

```diff
@@ -12,247 +13,77 @@ public RedisGraphAPITest() {
+    	ResultSet result = api.query("CREATE ({name:\"roi\",age:32})");
+    	Assert.assertFalse(result.hasNext());
+    	
+    	Assert.assertEquals("1", result.getStatistics().getStringValue(Label.NODES_CREATED));
+    	Assert.assertNull(result.getStatistics().getStringValue(Label.NODES_DELETED));
+    	Assert.assertNull(result.getStatistics().getStringValue(Label.RELATIONSHIPS_CREATED));
+    	Assert.assertNull(result.getStatistics().getStringValue(Label.RELATIONSHIPS_DELETED));
+    	Assert.assertEquals("2", result.getStatistics().getStringValue(Label.PROPERTIES_SET));
+    	Assert.assertNotNull(result.getStatistics().getStringValue(Label.QUERY_INTERNAL_EXECUTION_TIME));
     }
 
-    @org.testng.annotations.Test
-    public void testCreateLabeledNode() throws Exception {
+    @Test
+    public void testCreateLabeledNode() throws Exception {    	
         // Create a node with a label
-        RedisNode node = api.createLabeledNode("human","name", "roi", "age", 32);
-
-        // Validate node
-        Assert.assertNotNull(node.getId());
-        Assert.assertSame(node.getLabel(), "human");
-        Assert.assertSame(node.getAttributes().get("name"), "roi");
-        Assert.assertSame(Integer.parseInt(node.getAttributes().get("age")), 32);
-        Assert.assertSame(node.getAttributes().size(),2);
+    	ResultSet result = api.query("CREATE (:human{name:\"danny\",age:12})");
+    	Assert.assertFalse(result.hasNext());
+    	
+    	Assert.assertEquals("1", result.getStatistics().getStringValue(Label.NODES_CREATED));
+    	Assert.assertEquals("2", result.getStatistics().getStringValue(Label.PROPERTIES_SET));
+    	Assert.assertNotNull(result.getStatistics().getStringValue(Label.QUERY_INTERNAL_EXECUTION_TIME)); 
     }
 
-    @org.testng.annotations.Test
+    @Test
     public void testConnectNodes() throws Exception {
         // Create both source and destination nodes
-        RedisNode src = api.createNode("name", "roi", "age", 32);
-        RedisNode dest = api.createNode("name", "amit", "age", 30);
-
-        // Connect source and destination nodes.
-        RedisEdge edge = api.connectNodes(src, "knows", dest, "strength", "3", "from", "high-school");
-
-        // Validate edge
-        Assert.assertNotNull(edge.getId());
-        Assert.assertEquals(edge.getRelation(), "knows");
-        Assert.assertSame(edge.getAttributes().size(), 2);
-        Assert.assertSame(edge.getAttributes().get("strength"), "3");
-        Assert.assertSame(edge.getAttributes().get("from"), "high-school");
-        Assert.assertSame(edge.getSrc(), src);
-        Assert.assertSame(edge.getDest(), dest);
+    	ResultSet createResult1 = api.query("CREATE (:person{name:'roi',age:32})");
+    	ResultSet createResult2 = api.query("CREATE (:person{name:'amit',age:30})");
+    	
+    	// Connect source and destination nodes.
+    	ResultSet matchResult = api.query("MATCH (a:person), (b:person) WHERE (a.name = 'roi' AND b.name='amit')  CREATE (a)-[knows]->(a)");
```

Comment:
Rename matchResult, maybe resultset ?

Reply:
It's just following the rest of the results sets naming according to the query running